app/scripts/lib/stores/extension-store.test.ts

```diff
@@ -0,0 +1,95 @@
+import browser from 'webextension-polyfill';
+import ExtensionStore from './extension-store';
+
+const MOCK_STATE = { data: {}, meta: { version: 1 } };
+
+jest.mock('webextension-polyfill', () => ({
+  runtime: { lastError: null },
+  storage: { local: true },
+}));
+
+const setup = (
+  options: { localMock?: { get?: unknown; set?: unknown } | false } = {},
+) => {
+  if (typeof options.localMock === 'undefined') {
+    browser.storage.local =
+      jest.fn() as unknown as browser.Storage.LocalStorageArea;
+  } else if (options.localMock === false) {
+    browser.storage.local =
+      undefined as unknown as browser.Storage.LocalStorageArea;
+  } else {
+    browser.storage.local =
+      options.localMock as unknown as browser.Storage.LocalStorageArea;
+  }
+  return new ExtensionStore();
+};
+describe('ExtensionStore', () => {
+  beforeEach(() => {
+    global.sentry = {
+      captureException: jest.fn(),
+    };
```

Comment:
This is equivalent, except that it doesn't alter the global namespace and impact other tests.
```suggestion
    jest.replaceProperty(global.sentry, 'captureException', jest.fn());
```